keyboards/lets_split_eh/keymaps/resfury/keymap.c

```diff
@@ -0,0 +1,194 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _COLEMAK 0
```

Comment:
This is optional.

You may want to consider using an enum here, to make managing the layers easier.